tests/e2e/framework/framework.go

```diff
@@ -477,6 +493,11 @@ func (f *Framework) DeleteAll() error {
 			return fmt.Errorf("failed to delete namespace %q: %v", name, err)
 		}
 	}
+	/* Delete single pod in default namespace */
```

Comment:
Why are you trying to delete a single pod within the framework? I think I just missed the sense here but if not this should be moved from the framework to a test or what do you think?

Reply:
The `DeleteAll` function cleans up the cluster including created namespaces and pods. The default namespace is ignored for this cleanup. Since the `default` ServiceAccount in the default namespace plays a special role, this case this tested with a pod in the default namespace. Thus, this pod has to be cleaned up separately.